### DISCUSSION TITLE
Official Closure of Forumotion Channels

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#331] [Announcements]
As the title suggests, I have finally closed the old [NZ:P forum](https://nzportable.forumotion.com). 

What this means:
* Users can no longer post any new threads or reply to any existing threads on Forumotion
* New registrations are no longer allowed

What this does **not** mean:
* Content on Forumotion is inaccessible/hidden
* The site will be deleted

This change has been made now that all available community map content has been archived here on Discussions. I've taken the liberty of activating the remaining 4,300 pending users as well, just so everything could be cleaned up on that end.

I know quite a few people from back in the day are still here with us now, so to you all I'd like to say thanks for taking part in the discussions had in this community for so many years. Forumotion was definitely a neat hub, with a lot of personalization opportunity, but the required monetization via banners, forced URL shortening, and manual account activation requirements made it a pain to manage. 

Feel free to leave comments or questions in this thread to be answered.
Thanks.

**Comment by MisaelJama** [comment]
Could you download maps from forumotion and send them to me please, I beg you?🥺

**Reply by MotoLegacy** [reply]
> This change has been made now that all available community map content has been archived here on Discussions

They're already on here.

**Reply by MisaelJama** [reply]
The one from Castle of Doom is there but not for PSP, please can you get it for me?

**Reply by MotoLegacy** [reply]
Anything not here was because the links were dead and the content is lost.

**Reply by MisaelJama** [reply]
To please
